ui/app/common/obs/models/observation.js

```diff
@@ -54,7 +56,15 @@ Bahmni.Common.Obs.Observation = (function () {
             var displayValue = this.getDisplayValue();
             return this.isFileNameOfNewFormat() ? displayValue.substring(displayValue.lastIndexOf(this.getNewFormatFileNameSeparator()) + this.getNewFormatFileNameSeparator().length) : displayValue;
         },
-
+        getChiefComplaintCodedComment: function () {
+            if (this.isConceptNameChiefComplaintData()) {
+                return this.groupMembers[0].comment;
+            }
```

Comment:
It's generally better to return something (can be false or undefined) for scenarios when the condition fails to avoid unnecessary side effects.